docs/se/health/01_health.adoc

```diff
@@ -43,10 +43,15 @@ A typical health check combines the statuses of all the dependencies that
 
 === Prerequisites
 
-Declare the following dependency in your project:
+Declare the following dependencies in your project:
```

Comment:
This is not needed - `helidon-health` transitively depends on `helidon-media-jsonp`